homeassistant/components/overkiz/sensor.py

```diff
@@ -475,6 +475,22 @@ class OverkizSensorDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_REGULATION_MODE,
+        icon="mdi:auto-mode",
+        name="Regulation mode",
```

Comment:
What are the possible values for regulations mode? Do you have your diagnostics for this device by any chance?

You can't use `SensorDeviceClass.ENUM` without the options parameters.

>The sensor has a limited set of (non-numeric) states. The options property must be set to a list of possible states when using this device class.

Reply:
My investigation lead me to this list that I believe is exhaustive.

- **increase**: device is heating to reach target temperature
- **standby**: device is maintaining target temperature
- **none**: device is not heating

Reply:
Here is it (you made me discover this hass feature :) )
```
              {
                "type": "DiscreteState",
                "values": [
                  "increase",
                  "none",
                  "standby"
                ],
                "qualifiedName": "core:RegulationModeState"
              },
```

I see what you mean
- `increase` could be mapped to `HEATING`
- `none` could be mapped to `IDLE` (if device is on)
but I don't think there is a relevant mapping for `standby` though

Reply:
Could this not be mapped like:

| HVACAction Enum | String Value | 
|----------------|--------------| 
| HVACAction.OFF | "none" |
 | HVACAction.IDLE | "standby" |
 | HVACAction.HEATING | "increase" |

Reply:
I tried this mapping locally for a few days. It looks relevant enough indeed 👍
You can now review my changes

Reply:
Awesomes, thanks! Good to have these mapped in a native way.